internal/vulnstore/postgres/get.go

```diff
@@ -11,20 +14,34 @@ import (
 
 	"github.com/jackc/pgx/v4"
 	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/rs/zerolog"
 )
 
 func get(ctx context.Context, pool *pgxpool.Pool, records []*claircore.IndexRecord, opts vulnstore.GetOpts) (map[int][]*claircore.Vulnerability, error) {
-	// build our query we will make into a prepared statement. see build func definition for details and context
+	log := zerolog.Ctx(ctx).With().
+		Str("component", "vulnstore.get").
+		Logger()
+	// Build our query we will make into a prepared statement. See build func
+	// definition for details and context.
 	query, dedupedMatchers, err := getBuilder(opts.Matchers)
+	if err != nil {
+		return nil, err
+	}
+	h := fnv.New64a()
+	if _, err := io.WriteString(h, query); err != nil {
+		return nil, err
+	}
+	name := hex.EncodeToString(h.Sum(nil))
+	log.Debug().Str("name", name).Msg("built query")
```

Comment:
Is there value in displaying a hash of the query?

Reply:
I put it in there as assurance that a query is changing with a different set of matchexps provided